spec/spec_helper.rb

```diff
@@ -14,6 +14,8 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 
+require './spec/support/helpers/authentication'
```

Comment:
rails helper should already be including any file inside `support` directory. https://github.com/mmmmaximus/rails-truck-logistics/blob/main/spec/rails_helper.rb#L23
so you might be able to delete this